packages/itwinui-css/src/menu/menu.scss

```diff
@@ -40,6 +40,9 @@ $iui-active-outline: thin solid t(iui-color-foreground-primary);
   cursor: pointer;
   box-sizing: border-box;
   line-height: normal;
+  @include themed {
+    color: t(iui-text-color);
```

Comment:
This variable and the themed mixin are both removed in v1, so it doesn't make sense to release this fix to `main` branch, especially since it's not urgent

Reply:
Jon/Adam should check that the issue is fixed in v1, and if it is then we can label the issue as ready for release and not merge this.

Reply:
I agree with Mayank that this issue should be addressed after v1 is merged into `main`

Reply:
So use css variable and merge? Its new addition, not something we break.

Reply:
This has been changed

Reply:
there is no `iui-text-color` in variables package so this would still cause conflicts